common/src/main/java/org/apache/sedona/common/raster/RasterBandAccessors.java

```diff
@@ -39,6 +40,44 @@ public static Double getBandNoDataValue(GridCoverage2D raster) {
         return getBandNoDataValue(raster, 1);
     }
 
+    public static int getCount(GridCoverage2D raster, int band, boolean excludeNoDataValue) {
```

Comment:
This function should return an integer since max value can be 64 bit assuming 2 32 bit integers

Reply:
This is a very good point